node/service/src/chain_spec/moonbase.rs

```diff
@@ -238,6 +238,7 @@ pub fn testnet_genesis(
 				})
 				.collect(),
 		},
+		..Default::default()
```

Comment:
here both pallet_treasury instances are set to default, like here https://github.com/paritytech/polkadot/blob/master/node/service/src/chain_spec.rs#L286 and everywhere else I looked in substrate